Microsoft.AVS.Management/Microsoft.AVS.Management.psm1

```diff
@@ -284,15 +284,15 @@ function Get-CertificateFromServerToLocalFile {
             catch {
                 throw "The FQDN $($ResultUrl.Host) cannot be resolved to an IP address. Make sure DNS is configured."
             }
-
+            Write-Host "Connectivity to $($ResultUrl.Host) is verified."
```

Comment:
Let's indicate in the message what exactly we did here, which was not connectivity - it was name resolution.